### DISCUSSION TITLE
rtos arm configures the http server

### DISCUSSION THREAD

**Discussion by evercf** [cesanta/mongoose#2186] [Ask questions, get answers] [answered]
I would like to ask, if the serial port of arm is connected to the WiFi module, the ap mode is set on WiFi, and the http server is created, the computer is connected to the WiFi module, how to request the http server on arm? I used the code on the http example, it seems that the computer website tried to open the ip on WiFi, but there was no response

**Comment by scaprile** [comment, marked as answer]
I'm sorry, I can't understand you. You seem to be asking a basic routing question and not being able to configure your own network and this is not the place to ask that.
